tests/PHPUnit/Domain/Service/Precondition/StagingDirDoesNotExistUnitTest.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+
+namespace PhpTuf\ComposerStager\Tests\PHPUnit\Domain\Service\Precondition;
+
+use PhpTuf\ComposerStager\Domain\Service\Filesystem\FilesystemInterface;
+use PhpTuf\ComposerStager\Domain\Service\Precondition\StagingDirDoesNotExist;
+use PhpTuf\ComposerStager\Domain\Value\Path\PathInterface;
+use PhpTuf\ComposerStager\Tests\PHPUnit\TestCase;
+use Prophecy\Argument;
+
+/**
+ * @coversDefaultClass \PhpTuf\ComposerStager\Domain\Service\Precondition\StagingDirDoesNotExist
+ *
+ * @covers \PhpTuf\ComposerStager\Domain\Service\Precondition\StagingDirDoesNotExist::__construct
+ *
+ * @uses \PhpTuf\ComposerStager\Domain\Service\Precondition\AbstractPrecondition
+ *
+ * @property \PhpTuf\ComposerStager\Domain\Value\Path\PathInterface|\Prophecy\Prophecy\ObjectProphecy $activeDir
+ * @property \PhpTuf\ComposerStager\Domain\Value\Path\PathInterface|\Prophecy\Prophecy\ObjectProphecy $stagingDir
+ * @property \PhpTuf\ComposerStager\Domain\Service\Filesystem\FilesystemInterface|\Prophecy\Prophecy\ObjectProphecy $filesystem
+ */
+class StagingDirDoesNotExistUnitTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        $this->activeDir = $this->prophesize(PathInterface::class);
+        $this->activeDir
+            ->resolve()
+            ->willReturn(self::ACTIVE_DIR);
+        $this->stagingDir = $this->prophesize(PathInterface::class);
+        $this->stagingDir
+            ->resolve()
+            ->willReturn(self::STAGING_DIR);
+        $this->filesystem = $this->prophesize(FilesystemInterface::class);
+        $this->filesystem
```

Comment:
Same question. This seems like an incomplete prophecy; if it's actually needed, maybe a comment would be useful here.